src/System.Private.CoreLib/shared/System/Runtime/InteropServices/UnknownWrapper.cs

```diff
@@ -2,6 +2,7 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+#nullable enable
```

Comment:
constructor doesn't guard against obj being null

Reply:
It didn't seem like it made sense for `obj` to be null, but looking again I found one use where `null` is explicitly passed in on desktop, so I'll make both `obj` and `WrappedObject` nullable.

Reply:
This would make sense if an `out` semantic was required. I can't think of any other reason to permit null. @stephentoub can you provide a Framework example?

Reply:
@AaronRobinsonMSFT, e.g. https://referencesource.microsoft.com/#mscorlib/system/runtime/compilerservices/iunknownconstantattribute.cs,23